fdbbackup/backup.actor.cpp

```diff
@@ -584,8 +584,8 @@ CSimpleOpt::SOption g_rgRestoreOptions[] = {
 	{ OPT_RESTORE_TIMESTAMP,            "--timestamp",         SO_REQ_SEP },
 	{ OPT_KNOB,            "--knob_",          SO_REQ_SEP },
 	{ OPT_RESTORECONTAINER,"-r",               SO_REQ_SEP },
-	{ OPT_PREFIX_ADD,      "-add_prefix",      SO_REQ_SEP },
-	{ OPT_PREFIX_REMOVE,   "-remove_prefix",   SO_REQ_SEP },
+	{ OPT_PREFIX_ADD,      "--add_prefix",     SO_REQ_SEP },
+	{ OPT_PREFIX_REMOVE,   "--remove_prefix",  SO_REQ_SEP },
```

Comment:
Since this is a patch release, do we need to still support the single hyphen in order to not break users?

Reply:
Once this is merged to master I'll do another PR to remove the old versions.